dataland-backend/src/main/resources/application.properties

```diff
@@ -23,3 +23,8 @@ spring.datasource.url=jdbc:postgresql://backend-db:5432/backend
 spring.jpa.database-platform=org.hibernate.dialect.PostgreSQLDialect
 spring.jpa.show-sql=false
 spring.jpa.hibernate.ddl-auto=update
+spring.jackson.deserialization.fail-on-unknown-properties=true
```

Comment:
A few of them seem unused, actually.

Reply:
Which ones exactly? The "spring.jackson.deserialization.fail-on-unknown-properties" makes it so that when you send additional fields to Dataland, a 400 error is returned instead of it silently deleting the unknown field.